docs/guides/jwt-verification.md

```diff
@@ -0,0 +1,227 @@
+---
+title: JWT Verification
+lang: en-US
+meta:
+  - name: keywords
+    content: pomerium identity-access-proxy envoy jwt
+description: >-
+  This example demonstrates how to verify the Pomerium JWT assertion header using Envoy.
+---
+
+# JWT Verification
+This example demonstrates how to verify the [Pomerium JWT assertion header](https://www.pomerium.io/reference/#pass-identity-headers) using [Envoy](https://www.envoyproxy.io/).
```

Comment:
Maybe just add a short blurb about why this is useful? (e.g. a way to assert user went through pomerium without manual JWT verification, or mTLS).
- legacy applications
- third party apps